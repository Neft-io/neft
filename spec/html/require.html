<link rel="require" href="require_subfile.html">

<unit name="scope"><div id="unit-scope"></div></unit>

<scope></scope>
<elema></elema>
<elemb></elemb>