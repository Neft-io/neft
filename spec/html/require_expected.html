<unit><div id="unit-scope"></div></unit><unit><div id="unit-elema"></div><unit><div id="unit-elemb"></div></unit></unit><unit><div id="unit-elemb"></div></unit>